ee/query-service/dao/sqlite/modelDao.go

```diff
@@ -48,7 +48,17 @@ func InitDB(dataSourceName string) (*modelDao, error) {
 		updated_at INTEGER,
 		data TEXT  NOT NULL,
 		FOREIGN KEY(org_id) REFERENCES organizations(id)
-	);`
+	);
+	CREATE TABLE IF NOT EXISTS personal_access_tokens (
```

Comment:
do we need indexes: 
 a unique index for token. And non-unique for user_id. 
As such  pat table may not grow beyond a few rows. So full scan can work bettter.

Reply:
Added UNIQUE for the token. For user_id, I am not worried about performance for the same reason as you mentioned, the table will be small.